go.sum

```diff
@@ -410,6 +410,8 @@ github.com/rogpeppe/go-internal v1.3.0/go.mod h1:M8bDsm7K2OlrFYOpmOWEs/qY81heoFR
 github.com/rogpeppe/go-internal v1.6.1/go.mod h1:xXDCJY+GAPziupqXw64V24skbSoqbTEfhy4qGm1nDQc=
 github.com/rogpeppe/go-internal v1.8.1 h1:geMPLpDpQOgVyCg5z5GoRwLHepNdb71NXb67XFkP+Eg=
 github.com/rogpeppe/go-internal v1.8.1/go.mod h1:JeRgkft04UBgHMgCIwADu4Pn6Mtm5d4nPKWu0nJ5d+o=
+github.com/rquadling/terraform-config-inspect v0.0.0-20230303172622-9dd357c8a974 h1:iUSNgwjav+yZfRPSGUyywed1cvQdJPw8HaDRR1TMfw0=
+github.com/rquadling/terraform-config-inspect v0.0.0-20230303172622-9dd357c8a974/go.mod h1:l8HcFPm9cQh6Q0KSWoYPiePqMvRFenybP1CH2MjKdlg=
```

Comment:
this comment posted yesterday for the upstream `terraform-config-inspect` libraries support on validation, doesn't look promising

https://github.com/hashicorp/terraform-config-inspect/pull/74#issuecomment-1491154387

would it make sense to use a fork?

Reply:
I'd be happy to maintain a fork, but I'm not an expert in Terraform or Go, but with help I'm happy to learn.

Reply:
I've [asked](https://github.com/hashicorp/terraform-config-inspect/issues/73#issuecomment-1508176141) if there is any other option of having enhancements being maintained as a community fork or branch.

For me, the biggest issue is about trust. I _COULD_ create the lib, but my experience in Go is not good enough to know if I've introduced something that will cause problems upstream. Having the package within the Hashicorp domain would, I hope, get a greater number of eyes on it and so better reviews, etc.

Another option is to resurrect https://github.com/terraform-docs/terraform-config-inspect.